pkg/controller/manager/sync_action.go

```diff
@@ -42,9 +42,11 @@ func runSync(ctx context.Context, cluster *scyllav1.ScyllaCluster, authToken str
 				if c.AuthToken != authToken {
 					actions = append(actions, &updateClusterAction{
 						cluster: &managerclient.Cluster{
-							ID:   c.ID,
-							Name: naming.ManagerClusterName(cluster),
-							Host: naming.CrossNamespaceServiceNameForCluster(cluster),
+							ID:                     c.ID,
+							Name:                   naming.ManagerClusterName(cluster),
+							Host:                   naming.CrossNamespaceServiceNameForCluster(cluster),
+							ForceNonSslSessionPort: true,
+							ForceTLSDisabled:       true,
```

Comment:
please link https://github.com/scylladb/scylla-operator/issues/1766 in both places